assign.py

```diff
@@ -275,12 +279,12 @@ def main():
                 procversion = wf_info["ProcessingVersion"]
 
         # reading xrootd and secondary_xrootd values
-        if options.xrootd:
+        if options.xrootd is not None:
```

Comment:
I checked the--no_secondary_xrootd option for some running ACDCs, of course, without performing the assignment.
In the case of --no_xrootd, I haven't found an ACDC where the original request has been assigned over xrootd. But, as they function the same way, I see no problem.